frontend/src/containers/App.js

```diff
@@ -155,7 +155,8 @@ const App = () => {
 
             // Chart disabled notifications
             if (
-              error.response.request.responseURL.includes('silentError=true')
+              error.response.request.responseURL.includes('silentError=true') ||
+              data.path.includes('/queryPaths') // supress errors also for queryPaths endpoint (when performing a search from the menu)
```

Comment:
pls extract that as constant and move it in the same file where the API function which is calling that endpoint it is.

Reply:
I added the constant but I put it under constants where it should be and then imported it into App.js and the api file.